service/frontend/workflowHandler.go

```diff
@@ -2174,6 +2179,10 @@ func (wh *WorkflowHandler) SignalWithStartWorkflowExecution(ctx context.Context,
 		return nil, wh.error(err, scope)
 	}
 
+	searchattribute.SetType(
```

Comment:
Nit: Would be nice to avoid code duplication, but I understand that requires introducing an interface, which is probably not worth it.

Reply:
It is already just one func. Or are you talking about combining `SignalWithStartWorkflowExecution` with `StartWorkflowExecution`? Anyway, there is no easy way to fix it. I'd leave it as is for now.